traffic_ops/testing/api/v1/todb_test.go

```diff
@@ -333,6 +333,7 @@ func Teardown(db *sql.DB) error {
 	DELETE FROM profile;
 	DELETE FROM parameter;
 	DELETE FROM profile_parameter;
+	DELETE FROM topology;
```

Comment:
Because if you run the APIv1 against an "un-clean" database with DSes that have topologies, the tests will to clean up DSes because it violates a foreign key constraint.